recipes/smplotlib/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "smplotlib" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AstroJacobLi/smplotlib/releases/download/{{ version }}/smplotlib-{{ version }}.tar.gz
+  url: https://github.com/AstroJacobLi/smplotlib/archive/v{{ version }}.tar.gz
+  sha256: 677fd8939e7b0384ae7e2c3d6f7c6135bc9f2b1b87cfc5c9723209d7d32f8836
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=42.0
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - scipy
+    - matplotlib >=3.3.4
```

```suggestion
    - matplotlib-base >=3.3.4
```
Looks like this package is just a style set for matplotlib. Thus, it could be used headlessly and doesn't require QT.